docs/general/emulator-support-and-issues.md

```diff
@@ -29,10 +29,13 @@ This page focuses on supported emulators. For extensive notes on unsupported emu
   - Some boards may not be fully exposed.
 - ✅ libretro core: **Flycast**
   - Used for Atomiswave and NAOMI 1/2.
+- ✅ Standalone emulator: **[Flycast](https://github.com/flyinghead/flycast/releases)**
```

Comment:
that links to and old release

refer: https://github.com/RetroAchievements/docs/pull/217

Reply:
Most of our players using Flycast are on Windows. The repo's README.md effectively directs Windows users to go to the builds page, which is for downloading nightlies.

Is it desirable to have users download nightly releases over the 2.4 versioned release?

Reply:
I'm going to approve + merge this for now. Let's follow-up in https://github.com/RetroAchievements/docs/pull/217.